scripts/postinstall.sh

```diff
@@ -6,13 +6,13 @@
 
 if [ "$SKIP_SERVICES" != "true" ]; then
 
-  SERVICES=( paywall smart-contracts unlock-app locksmith tests tickets newsletter wedlocks unlock-js unlock-protocol.com docker/development )
-
+  SERVICES=( smart-contracts tests tickets wedlocks unlock-js unlock-protocol.com docker/development )
+  ROOT_DIR=$(pwd)
```

Comment:
There was a bug before where in the last iteration of this loop we would be in `docker/development` and then `cd ..` leaving us in `docker/`, so the later `rm` commands wouldn't do anything. Always `cd`ing to a full path fixes that.